packages/botonic-plugin-google-translate/README.md

```diff
@@ -0,0 +1,65 @@
+# Botonic Plugin Google Translate
+
+## What does this plugin do?
+This plugin uses [Google Cloud Translation API](https://cloud.google.com/translate) to detect the language of the user input text.
+
+## Setup
+
+### Install the plugin
+To install this plugin, you only need to execute the following command:
+
+```bash
+npm i @botonic/plugin-google-translate 
+```
+
+### Enable Cloud Translation API
+Before using this plugin, you need a project that has the Cloud Translation API enabled and its credentials. See [Cloud Translation Set Up Guide](https://cloud.google.com/translate/docs/setup).
```

Comment:
what do you mean by a project? Is it enough to sat that you need to setup  Cloud Translation API credentials?